Migrations/migrations.sql

```
@@ -0,0 +1,243 @@
+    ALTER TABLE [dbo].[orders]  WITH CHECK ADD CONSTRAINT [CK_orders_status] CHECK  (([status]='delivered' OR [status]='rejected' OR [status]='accepted'))
+GO
+
+ALTER TABLE dbo.orders
+DROP CONSTRAINT CK_orders_mode_of_payment;
+
+ALTER TABLE dbo.orders
+ALTER COLUMN mode_of_payment bit;
+
+
+
+
```

SQLFluff (4.2.2)

[error] Too many consecutive blank lines.

(LT15)